tutorials/getting-started/pushing-assets.md

```diff
@@ -81,6 +81,14 @@ This will push the file `lib/modules/apostrophe-assets/public/js/site.js` to the
 
 > Want to use `gulp`, `browserify`, `grunt` and friends? Go for it! Just set up your configuration so that the compiled output file is pushed by Apostrophe. Apostrophe doesn't need to know you are using these tools for you to be successful with them. Separation of concerns is a good thing.
 
+## Including webfonts, images, and other assets
+
+We know you might not want to stick with the fonts that come with apostrophe, so we've accounted for this. Include any assets you want to use on your site in `lib/modules/apostrophe-assets/public/`. In your less files and nunjucks templates you'll then look for the assets in `/modules/my-apostrophe-assets/`. This might not be very intuitive, but it allows symbolic links from `public/modules` to differentiate between the npm version of a module and additions you've made at a project-level. 
```

Comment:
The main point not made here:

You can do this in any module, and doing it in your own thematically appropriate module can make for nicer namespacing. It also eliminates the need for the "my-" prefix. So I would say it's a best practice to use your own module. The important thing is to do it *in a module* so the bundler takes it into account. So I'd start there rather than ending there (I see you did mention it at the very end).

Finally it is worth mentioning that technically this matters only when creating asset bundles for the cloud, but since you may eventually decide to do that, it's a worthwhile practice to learn.

Reply:
All good points. Would you prefer these changes take place in the how-to for apostrophe in the cloud then? I originally had it there, but moved it here when I realized that you _can_ do this at any time, regardless of if you're integrating with the cloud yet or not.